Packs/TaniumThreatResponse/Integrations/TaniumThreatResponseV2/TaniumThreatResponseV2.py

```diff
@@ -424,7 +420,7 @@ def test_module(client, data_args):
         raise ValueError(f'Please check your credentials and try again. Error is:\n{str(e)}')
 
 
-def fetch_incidents(client, alerts_states_to_retrieve, label_names_to_retrieve, last_run, fetch_time, max_fetch):
+def fetch_incidents(client, alerts_states_to_retrieve, label_name_to_retrieve, last_run, fetch_time, max_fetch):
```

Comment:
label_names_to_retrieve isnt a list anymore? What changed? Lets add type refs

Reply:
It's not anymore - the issue is divided into two parts:
1. The integration didn't know how to handle cases where empty label names were given.
2. Unlike what was written in the first place, the API can handle only one label at a time.
that's why label_names_to_retrieve > label_name_to_retrieve

Reply:
I'll add type refs & unit tests.

Reply:
is that a change that the api made? Or we just implemented this wrong in the first place?